REQUIRE

```diff
@@ -1,7 +1,8 @@
 julia 0.5
-DataStructures 0.4.3
+DataStructures 0.5.0
 DataFrames 0.9
-NullableArrays 0.0.10
-CategoricalArrays 0.0.6
+NullableArrays 0.1.0
+CategoricalArrays 0.1.0
+AxisArrays 0.0.6
```

Comment:
So you've decided to use AxisArrays rather than NamedArrays? I'm still unsure what's the best approach (e.g. for my package FreqTables). The generality of `AxisArray` (numeric and categorical axes) is appealing, but the goal of the package does not seem to completely match the R approach to array names. In particular, the printing of names before the data (rather than on each line as in R/NamedArrays) isn't practical for categorical axes.

@mbauman What do you think about this? Are the goals compatible? Could the printing evolve towards what NamedArrays does? i.e. something like this:
```julia
4×4 Named Array{Int64,2}
Dim1 ╲ Dim2 │  A   B   C   D
────────────┼───────────────
a           │ 30  20  30  20
b           │ 30  20  30  20
c           │ 20  30  20  30
d           │ 20  30  20  30
```